WcaOnRails/app/views/delegate_reports/show.html.erb

```diff
@@ -29,6 +29,6 @@
   <%= render @delegate_report %>
 
   <% if current_user.can_edit_delegate_report?(@competition.delegate_report) %>
-    <%= link_to icon("pencil", "Edit"), delegate_report_edit_path(@competition), class: "btn btn-default" %>
+    <%= link_to icon("pencil", "Edit Report"), delegate_report_edit_path(@competition), class: "btn btn-default" %>
```

Comment:
Changed to make the test easier to write (there's another link on the page with the text "Edit"), but I think it's a good change anyways.